### PR TITLE
New package: Samsung.SWUpdate version 2.1.14.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111898&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SWUpdate
+PackageVersion: 2.1.14.1
```
Comment (Trenly) [inline manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.installer.yaml:5]:
```suggestion
PackageVersion: 2.1.14
```

File: manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SWUpdate
+PackageVersion: 2.1.14.1
```
Comment (Trenly) [inline manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.1.14
```

File: manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SWUpdate
+PackageVersion: 2.1.14.1
```
Comment (Trenly) [inline manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.yaml:5]:
```suggestion
PackageVersion: 2.1.14
```

File: manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SWUpdate
+PackageVersion: 2.1.14.1
+PackageLocale: en-US
+Publisher: Samsung
```
Comment (Trenly) [inline manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml:7]:
```suggestion
Publisher: Samsung Electronics CO., LTD.
```

File: manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SWUpdate
+PackageVersion: 2.1.14.1
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Software
```
Comment (Trenly) [inline manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml:8]:
```suggestion
PackageName: SW Update
```

File: manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SWUpdate
+PackageVersion: 2.1.14.1
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Software
+License: Proprietary
+ShortDescription: Software drivers for varius Samsung computers
```
Comment (Trenly) [inline manifests/s/Samsung/ATIV-SW/Book9/2.1.14.1/Samsung.ATIV-SW.Book9.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Software drivers for various Samsung computers
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by Trenly** [comment]
[Policy] Needs-Author-Feedback

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**